mitosheet/setup.py

```diff
@@ -119,8 +119,13 @@
         extras_require = {
             'test': [
                 'pytest',
+                'pytest-mock',
```

Comment:
We pay much of the cost of dependencies in terms of long-term complexity and maintaince, even when we only take them on as test dependencies. Moreover, I do not think that we need a full additional library to mock functions - there is Python built-in support for it.

Going forward, let's put _all_ dependencies through our dependency review process, even if they are just test/dev dependencies. For this one, I think writing this ourselves is <25 lines of code -- so let's def just do that!

Can you look into mocking functions without taking on additional dependencies here? You might be able to just use the unittest.mock.patch function, or potentially the `@patch` decorator. In either case, feel free to build some testing utilities that make it easier to mock in the future (potentially with the same interface as the current library you're using).